templates/default/stunnel.conf.erb

```diff
@@ -63,6 +63,11 @@ debug = <%= node['stunnel']['debug'] %>
 output = <%= node['stunnel']['output'] %>
 <% end -%>
 
+; Enable/disable FIPS mode
+<% unless node['stunnel']['fips'].nil? %>
```

Comment:
That makes sense and looking further down the file it appears we've got that pattern established down there too so the unless/nil? check can stay. I'll still request that you move the comment within that block and add the whitespace removal at the end of the tag.